server/pom.xml

```diff
@@ -262,6 +262,11 @@
             <version>2.6</version>
             <scope>compile</scope>
         </dependency>
+        <dependency>
+            <groupId>com.codahale.metrics</groupId>
+            <artifactId>metrics-core</artifactId>
```

Comment:
@OneSizeFitsQuorum see this.  ratis need this jar?

Reply:
In my computer, when I run UT test related to ratis consensus protocol, it tells me I miss this jar.

Reply:
seems weird